components/event-sources/reconciler/httpsource/reconcile_test.go

```diff
@@ -105,9 +113,7 @@ func TestReconcile(t *testing.T) {
 				NewHTTPSource(tNs, tName),
 			},
 			WantCreates: []runtime.Object{
-				NewChannel(tNs, tName,
```

Comment:
I moved all the redundant calls to `NewXYZ()` to helper funcs. It should make the tests slightly shorter and more readable.